src/kernels/jupyter/launcher/jupyterExecution.ts

```diff
@@ -210,9 +205,6 @@ export class JupyterExecutionBase implements IJupyterExecution {
                 cancelToken
             );
         } else {
-            // Prepare our map of server URIs
-            await this.jupyterConnection.updateServerUri(options.uri);
-
             // If we have a URI spec up a connection info for it
             return this.jupyterConnection.createConnectionInfo(options.uri);
```

Comment:
`cerateConnectionInfo` will ensure the server Uri has been updated, calling code need to know about such internals